eegnb/experiments/visual_p300/p300.py

```diff
@@ -20,18 +21,18 @@ def __init__(self, duration=120, eeg: EEG=None, save_fn=None,
         super().__init__(exp_name, duration, eeg, save_fn, n_trials, iti, soa, jitter)
         
     def load_stimulus(self):
-        load_image = lambda fn: visual.ImageStim(win=mywin, image=fn)
+        
+        load_image = lambda fn: visual.ImageStim(win=self.mywin, image=fn)
         # Setup graphics
-        mywin = visual.Window([1600, 900], monitor="testMonitor", units="deg", fullscr=True)
-        targets = list(map(load_image, glob(os.path.join(CAT_DOG, "target-*.jpg"))))
-        nontargets = list(map(load_image, glob(os.path.join(CAT_DOG, "nontarget-*.jpg"))))
+        self.targets = list(map(load_image, glob(os.path.join(CAT_DOG, "target-*.jpg"))))
+        self.nontargets = list(map(load_image, glob(os.path.join(CAT_DOG, "nontarget-*.jpg"))))
         
-        return [nontargets, targets]
+        return [self.nontargets, self.targets]
 
-    def present_stimulus(self):
+    def present_stimulus(self, ii):
```

Comment:
I think `idx` is a better name than `ii` (which I've seen used for vectors), type annotation would also help :)

Reply:
I agree, done